Bottom: 7709b4535e217d082976efc6095e0b6108adb8de
Top:    4ea9c531bb71f49dfa3a2778fe4ab49d5dd3df1f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-12 19:08:01 +0000

Refresh of convert-layer-and-group

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 70cc51b..fb6c0a0 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -152,8 +152,7 @@ float colors[12][3] = {{1., 0., 0.},
 #define CIRC_SEGS 64
 
 static void
-draw_quad_edge (edge_ref e,
-                G_GNUC_UNUSED void *data)
+draw_quad_edge (edge_ref e, void *data)
 {
 #if 0
   int id = ID(e) % 12;
@@ -198,8 +197,7 @@ draw_quad_edge (edge_ref e,
 }
 
 static void
-object3d_draw_debug_single (object3d *object,
-                            G_GNUC_UNUSED void *user_data)
+object3d_draw_debug_single (object3d *object, void *user_data)
 {
   g_return_if_fail (object->edges != NULL);
 
diff --git a/src/polygon.c b/src/polygon.c
index ab4e944..f587028 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -354,7 +354,7 @@ original_poly (PolygonType * p)
   POLYAREA *np = NULL;
   Cardinal n;
   Vector v;
-  unsigned int hole = 0;
+  int hole = 0;
 
   if ((np = poly_Create ()) == NULL)
     return NULL;
